browser-test/src/application_csv_download.test.ts

```diff
@@ -1,35 +1,40 @@
 import { startSession, logout, loginAsGuest, loginAsAdmin, ApplicantQuestions, AdminQuestions, AdminPrograms, endSession } from './support'
 
 describe('normal application flow', () => {
-  // If this times out, a likely cause is the .click() calls in
-  // support/admin_programs, which are called out as being asserts.
-  jest.setTimeout(25000);
+  // Our browser tests could be long-running (as of 2021-04-06, the longest
+  // takes 27 seconds).
+  jest.setTimeout(200000);

```

Comment:
Question (blocking): can we set this globally rather than per-test file if we need it? Also can we set it to a lower value maybe just double the longest test?

Reply:
Good call.

Reply:
Done.